src/gui/screen_printing_serial.cpp

```diff
@@ -10,8 +10,38 @@
 #include "stm32f4xx_hal.h" //HAL_GetTick
 #include "screens.h"
 
+#define BUTTON_TUNE       0
+#define BUTTON_PAUSE      1
+#define BUTTON_DISCONNECT 2
+
 #pragma pack(push)
-#pragma pack(1)
+
+enum item_id_t {
+    iid_tune,
+    iid_pause,
+    // iid_stop,
+    // iid_resume,
+    iid_disconnect,
+    iid_count
+};
+
+const uint16_t serial_printing_icons[iid_count] = {
+    IDR_PNG_menu_icon_settings,
+    IDR_PNG_menu_icon_pause,
+    // IDR_PNG_menu_icon_stop,
+    // IDR_PNG_menu_icon_resume,
+    IDR_PNG_menu_icon_stop // disconnect
+};
+
+const char *serial_printing_labels[iid_count] = {
+    "Tune",
+    "Pause",
+    // "Stop",
+    // "Resume",
+    "Disconnect"
+};
+
+#pragma pack(push, 1)
```

Comment:
and another unterminated #pragma pack - very dangerous leaving it unterminated with #pragma pack(pop)